database/query/profiles.sql

```diff
@@ -22,15 +22,25 @@ WHERE id = $1 AND project_id = $2 RETURNING *;
 INSERT INTO entity_profiles (
     entity,
     profile_id,
-    contextual_rules) VALUES ($1, $2, sqlc.arg(contextual_rules)::jsonb) RETURNING *;
+    contextual_rules,
+    migrated
+) VALUES (
+    $1,
+    $2,
+    sqlc.arg(contextual_rules)::jsonb,
+    FALSE
```

Comment:
This is redundant given that the default value is FALSE for this column, but I want to set it explicitly to remind myself to change it in the next PR.